### DISCUSSION TITLE
Trying to get UDP multicast working as input, no luck

### DISCUSSION THREAD

**Discussion by TheSashmo** [bluenviron/mediamtx#1744] [Questions]
I am running meidamtx latest release on mac ventura m1.

I am able to publish streams easily (thanks, its very easy), but when trying to add a multicast source to the path example

  udp:
    source: udp://239.1.2.58:1234

I get the the error:
INF [path udp] [udp source] ERR: setsockopt: can't assign requested address

I did a wireshark and tcpdump, and I don't even see the IGMP request go out on any of the network interfaces that I have.  

Suggestions?

**Comment by TheSashmo** [comment]
Also using the identical example of gstreamer gives me the same result.  I think its something to do with routes on the network and how I have one physical interface for internet and another for multicast.

**Comment by aler9** [comment]
Hello, i checked and it turned out that the server was returning an error when a single interface doesn't support multicast, while it should return an error only if ALL interfaces don't support multicast.
This is fixed by https://github.com/bluenviron/gortsplib/pull/293 and https://github.com/bluenviron/mediamtx/pull/1874

**Comment by aler9** [comment]
fixed in v0.23.4

[a comment by an automated account is not shown]
